demos/using-dev-build/README.md

```diff
@@ -31,6 +39,28 @@ At this point, the replica (for all practical matters, a local version of the In
 
 If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
 
+### Adding Internet Identity to your Local Project
+
+Add the following snippet to the `canister` section in your `dfx.json` file (see full example [here](https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json)):
+```json
+"internet_identity": {
+  "type": "custom",
+  "candid": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity.did",
+  "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm",
+  "remote": {
+    "id": {
+      "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
+    }
+  }
+}
+```
+
+This adds the [dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity to your (local) project. The `remote` property makes sure, that your project will _not_ create a copy of Internet Identity on the IC when deploying to production.
+
+Using the dev build locally makes test automation easy:
```

Comment:
Should we maybe start with that? i.e. explain why we're doing it before we actually do it